distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -61,4 +114,131 @@ public void testCswRegistryIngest() throws Exception {
 
     }
 
+    @Test
+    public void testCswRegistryCreate() throws Exception {
```

Comment:
This test has no assertions - it needs to be refactored to use an external interface and assert on the response objects themselves.